packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/new/new.html

```diff
@@ -1,4 +1,7 @@
-<div data-ovh-alert="app.dedicatedClouds.datacenter.backup.new"></div>
+<div
+    data-ovh-alert="app.dedicatedClouds.datacenter.backup.new"
+    id="backupNewMainAlert"
```

Comment:
The `id` is no longer required. Please take this change from the parent branch `feat/veeam-backup`

Reply:
it will be taken care during rebase/merge